tests/test_skill_index.py

```
@@ -0,0 +1,478 @@
+
+
+@dataclass
+class _FakeArtifact:
+    """Minimal SkillV1Artifact lookalike for testing without importing extensions."""
+
+    name: str
+    description: str
+    prompt_template: str
+    tools_used: list[str] = field(default_factory=list)
+    created_at: datetime = field(
+        default_factory=lambda: datetime.now(timezone.utc)
```

Ruff (0.15.10)

[warning] Use `datetime.UTC` alias

Convert to `datetime.UTC` alias

(UP017)